program/rust/src/instruction.rs

```diff
@@ -105,3 +108,44 @@ pub fn load_command_header_checked(data: &[u8]) -> Result<OracleCommand, OracleE
     }
     OracleCommand::from_i32(command_header.command).ok_or(OracleError::UnrecognizedInstruction)
 }
+
+#[repr(C)]
+#[derive(Zeroable, Pod, Copy, Clone)]
+pub struct AddPriceArgs {
+    pub header: CommandHeader,
+    pub expo_:  i32,
+    pub ptype_: u32,
+}
+pub type InitPriceArgs = AddPriceArgs;
+
+#[repr(C)]
+#[derive(Zeroable, Pod, Copy, Clone)]
+pub struct AddPublisherArgs {
+    pub header: CommandHeader,
+    pub pub_:   pc_pub_key_t,
+}
+
+pub type DelPublisherArgs = AddPublisherArgs;
+
+
+unsafe impl Pod for SetMinPubArgs {
+}
+/// This is embarassing, Pod derivation doesn't work for this struct because it has padding in the
+/// end WARNING: DO NOT ACCESS THE 3 LAST BYTES OF THIS STRUCT!
```

Comment:
last 2 bytes right?

also how about adding `pub unused_: u8` as the last field to get the Pod derivation to work?

Reply:
You're so right!